src/test/java/com/okta/tools/aws/settings/ConfigurationTest.java

```diff
@@ -1,246 +0,0 @@
-
-        configuration.addOrUpdateProfile(profileName, role_arn, region);
-        configuration.save(configurationWriter);
-
-        String expected = existingProfile + "\n\n" + manualRole;
-        String given = StringUtils.remove(configurationWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-
-    /*
-     * Test writing a new profile to an existing configuration file with custom prefix.
-     */
-    @Test
-    void addOrUpdateProfileToExistingConfigFileCustomPrexi() throws IOException {
-        final StringReader configurationReader = new StringReader(existingProfile);
-        final StringWriter configurationWriter = new StringWriter();
-        final Configuration configuration = new Configuration(configurationReader, environmentWithCustomPrefix);
-
-        configuration.addOrUpdateProfile(profileName, role_arn, region);
-        configuration.save(configurationWriter);
-
-        String expected = existingProfile + "\n\n" + manualRoleCustomPrefix;
-        String given = StringUtils.remove(configurationWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-
-    /*
-     * Test writing a new default profile to an existing configuration file.
-     */
-    @Test
-    void addOrUpdateDefaultProfileToExistingConfigFile() throws IOException {
-        final StringReader configurationReader = new StringReader(existingProfile);
-        final StringWriter configurationWriter = new StringWriter();
-        final Configuration configuration = new Configuration(configurationReader);
-
-        configuration.addOrUpdateProfile("default", role_arn, region);
-        configuration.save(configurationWriter);
-
-        String expected = existingProfile + "\n\n" + defaultRole;
-        String given = StringUtils.remove(configurationWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-
-    /*
-     *  Test updating an existing profile.
-     */
-    @Test
-    void addOrUpdateProfileToExistingProfile() throws IOException {
-        final StringReader configurationReader = new StringReader(manualRole + "\n\n" + existingProfile);
-        final StringWriter configurationWriter = new StringWriter();
-        final Configuration configuration = new Configuration(configurationReader);
-
-        final String updatedPrefix = "updated_";
-        final String expected = "[profile " + profileName + "]\n"
-                + Configuration.ROLE_ARN + " = " + updatedPrefix + role_arn + "\n"
-                + SOURCE_PROFILE + " = " + profileName + "_source\n"
-                + Configuration.REGION + " = " + region
-                + "\n\n" + existingProfile;
-
-
-        configuration.addOrUpdateProfile(profileName, updatedPrefix + role_arn, region);
-        configuration.save(configurationWriter);
-
-        String given = StringUtils.remove(configurationWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-}
```

Comment:
:fire: tests for writing ~/.aws/config which we no longer write, so we no longer needed to test.